homeassistant/components/metoffice/sensor.py

```diff
@@ -33,105 +37,121 @@
         entity_registry_enabled_default=True,
     ),
-    SensorEntityDescription(
+    MetOfficeSensorEntityDescription(
         key="temperature",
+        native_attr_name="screenTemperature",
         name="Temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         icon=None,
         entity_registry_enabled_default=True,
     ),
-    SensorEntityDescription(
+    MetOfficeSensorEntityDescription(
         key="feels_like_temperature",
+        native_attr_name="feelsLikeTemperature",
         name="Feels like temperature",
         device_class=SensorDeviceClass.TEMPERATURE,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         icon=None,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    MetOfficeSensorEntityDescription(
+        key="pressure",
+        native_attr_name="mslp",
+        name="Pressure",
+        device_class=SensorDeviceClass.ATMOSPHERIC_PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PA,
+        icon=None,
+        entity_registry_enabled_default=True,
+    ),
+    MetOfficeSensorEntityDescription(
         key="wind_speed",
+        native_attr_name="windSpeed10m",
         name="Wind speed",
-        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        native_unit_of_measurement=UnitOfSpeed.METERS_PER_SECOND,
         # Hint mph because that's the preferred unit for wind speeds in UK
         # This can be removed if we add a mixed metric/imperial unit system for UK users
         suggested_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
         device_class=SensorDeviceClass.WIND_SPEED,
         entity_registry_enabled_default=True,
     ),
-    SensorEntityDescription(
+    MetOfficeSensorEntityDescription(
         key="wind_direction",
+        native_attr_name="windDirectionFrom10m",
         name="Wind direction",
         icon="mdi:compass-outline",
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    MetOfficeSensorEntityDescription(
         key="wind_gust",
+        native_attr_name="windGustSpeed10m",
         name="Wind gust",
-        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        native_unit_of_measurement=UnitOfSpeed.METERS_PER_SECOND,
         # Hint mph because that's the preferred unit for wind speeds in UK
         # This can be removed if we add a mixed metric/imperial unit system for UK users
         suggested_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
         device_class=SensorDeviceClass.WIND_SPEED,
         entity_registry_enabled_default=False,
     ),
-    SensorEntityDescription(
+    MetOfficeSensorEntityDescription(
         key="visibility",
-        name="Visibility",
-        icon="mdi:eye",
-        entity_registry_enabled_default=False,
-    ),
-    SensorEntityDescription(
-        key="visibility_distance",
```

Comment:
Please properly list in the breaking change section which entities are being removed.